pom.xml

```diff
@@ -284,6 +284,17 @@ THE SOFTWARE.
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <dependencyManagement>
+        <dependencies>
+            <!-- TODO: Remove this when https://github.com/jenkinsci/subversion-plugin/pull/240 is fixed -->
```

Comment:
```suggestion
            <!-- TODO: Remove this when https://github.com/jenkinsci/subversion-plugin/pull/240 is released -->
```

which I hope to do today. Care to review https://github.com/jenkinsci/subversion-plugin/pull/241?

Reply:
> I hope to do today

Well, if that is going to happen so soon, we can completely avoid all these dances with workaround and _just wait_.

Reply:
Anyway, applied your suggestion.